tests/blackbox/stratis_cli_cert.py

```diff
@@ -113,6 +115,29 @@ def setUp(self):
         time.sleep(1)
         exec_command(["udevadm", "settle"])
 
+    def test_permissions(self, command_line, permissions, exp_stdout_empty):
+        """
+        Test running CLI commands with and without root permissions.
+        :param list command_line: The arguments needed to execute the Stratis command being tested
+        :type command_line: List of str
+        :param bool permissions: True if the Stratis command needs root permissions to succeed,
+                                    otherwise False.
+        :param bool exp_stdout_empty: True if stdout is expected to be empty
+                                        when Stratis command succeeds.
+        """
+
+        os.seteuid(_NON_ROOT)
+        try:
+            if permissions:
+                self.unittest_command(command_line, 1, False, True)
+            else:
+                self.unittest_command(command_line, 0, True, exp_stdout_empty)
+        except Exception as err:
+            os.seteuid(_ROOT)
+            raise err
+
+        self.unittest_command(command_line, 0, True, exp_stdout_empty)
```

Comment:
I recall now that I thought this should only run if ```permissions``` is True. Otherwise, the call, whatever it is, should already have succeeded, in which case running it twice may cause a test error.

Reply:
Ah, this makes sense, since unlike the dbus calls, the cli commands are not idempotent.

Reply:
I also noticed that with this arrangement, the euid does not get reset to root on successful runs of the try block.